proc_CPMG.py

```diff
@@ -24,8 +25,8 @@
     fl.image(s)
     #{{{select echo decay fit function
     s.ft('t2')
-    curve = plot_curve(s, None, f_range, curve='decay')
-    quit()
+    f,T2 = decay(s, f_range, indirect='tE')
+    fl.plot_curve(f,'T2 relaxation decay')
```

Comment:
two things here:

- I liked the idea of what you started here and am consolidating+organizing further by defining a fitting function and a function for plotting the fits
- Because `plot_curve` is a method of `fl_mod`, it's first argument is the `fl_mod` instance (usually called "self", though I left it called `fl`), and the function is called as a method, as here.